cdci_data_analysis/flask_app/app.py

```diff
@@ -259,6 +259,12 @@ def inspect_state():
     return state_data_obj
 
 
+@app.route('/instr-list')
```

Comment:
There are now two endpoints: `/api/instr-list` and `/instr-list` which could be misleading
If you intend to deprecate the one with 'api', probably add deprecation warning?

Reply:
probably we should directly remove `/api/instr-list`, and adapt oda_api in turn

Reply:
> There are now two endpoints: `/api/instr-list` and `/instr-list` which could be misleading If you intend to deprecate the one with 'api', probably add deprecation warning?

https://github.com/oda-hub/oda_api/pull/178

Reply:
do we want to keep the endpoint as `/api/instr-list` od we can remove the `/api` ?

Reply:
what do you think, what are the reasons in favor of each choice? Which one follows the approach we used in the rest of the application?

If you do https://github.com/oda-hub/oda_api/pull/178 it breaks compatibility. It absolutely has to have deprecation message and a redirection.

Reply:
redirection now takes place

Reply:
> redirection now takes place

wait, but concerning this:

> what do you think, what are the reasons in favor of each choice? Which one follows the approach we used in the rest of the application?

why did we need to change the endpoint removing the "/api/" ?

Reply:
To actually have it more aligned with the majority of endpoint names we implemented. But I don't really now the inital reasons why there was `/api`

Reply:
all of the analogous ones are still with `/api/`. But indeed the newer ones do not have it. We should change them all at once at some point. For here, what you added now is ok.